e2e/fixtures/initcontainers_ordertest_pod.yml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
```

Comment:
Can we combine the 2 pods/tests into one to save e2e test time? 
pod yaml file that contains:
- multiple initContainer
- InitContainer writes to a file and main container to verify that
- add volumes & volume mounts to the pod spec

Reply:
using one combined e2e test for initcontainers